.jest/filenameTransform.js

```diff
@@ -0,0 +1,7 @@
+const { basename } = require('path')
+
+module.exports = {
+  process(_src, filename) {
+    return 'module.exports = ' + JSON.stringify(basename(filename)) + ';'
```

Comment:
Unlike `fileMocks.js`, this allows us to distinguish between different asset files, i.e. test that the code loads the correct asset, not just any asset.